tests/end-to-end/external-solution-integration/internal/scenario/event_mesh/scenario.go

```diff
@@ -13,6 +13,7 @@ type Scenario struct {
 	skipSSLVerify     bool
 	applicationTenant string
 	applicationGroup  string
+	waitTime          int
```

Comment:
Does it make sense to change `int` to `time.Duration`.

Reply:
Done